StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/FileHeaderCodeFixProvider.cs

```diff
@@ -352,9 +353,13 @@ private static SyntaxTriviaList CreateNewHeader(string prefixWithLeadingSpaces,
 
         private static string WrapInXmlComment(string prefixWithLeadingSpaces, string copyrightText, string filename, StyleCopSettings settings, string newLineText)
         {
+            string encodedFilename = new XText(filename).ToString();
+            string encodedCompanyName = new XText(settings.DocumentationRules.CompanyName).ToString();
```

Comment:
Well I learnt something new today (Never thought about it before...) I'll fix and add quotes into the company name for the tests.